src/components/PostDetails.tsx

```diff
@@ -140,7 +82,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => handleDeleteComment(comment.id)}
```

Comment:
it is bad practice to make anonymous functions in JSX
better to create function in component and pass reference here

Reply:
2 commit with the following code